packages/office-addin-debugging/src/start.ts

```diff
@@ -172,72 +172,87 @@ export async function runPackager(commandLine: string, host: string = "localhost
     }
 }
 
-
 /**
- * Start debugging
- * @param options start debugging options.
+ * `startDebugging` options.
  */
-export async function startDebugging(options: {
+export interface StartDebuggingOptions {
+
     /**
      * The path to the manifest file.
      */
     manifestPath: string,
+
     /**
      * The type of application to debug.
      */
     appType: AppType,
+
     /**
      * The Office application to debug.
      */
     app?: OfficeApp,
+
     /**
      * The method to use when debugging.
      */
     debuggingMethod?: DebuggingMethod,
+
     /**
      * Specify components of the source bundle url.
      */
     sourceBundleUrlComponents?: devSettings.SourceBundleUrlComponents,
+
     /**
      * If provided, starts the dev server.
      */
     devServerCommandLine?: string,
+
     /**
      *  If provided, port to verify that the dev server is running.
      */
     devServerPort?: number,
+
     /**
      *  If provided, starts the packager.
      */
     packagerCommandLine?: string,
+
     /**
      * Specifies the host name of the packager.
      */
     packagerHost?: string,
+
     /**
      * Specifies the port of the packager.
      */
     packagerPort?: string,
+
     /**
-     * If false, start without debugging.
+     * Enable debugging.
```

Comment:
"If false, start without debugging; if true or undefined, start with debugging enabled."

Reply:
Please make sure that it continues to have the behavior to enable debugging if undefined.